### DISCUSSION TITLE
playlist lost on (re-)boot

### DISCUSSION THREAD

**Discussion by ghost** [rern/rAudio#69] [Fixed Issues]
Using rAudio-1 (202101021) on a Rpi 3b+ and allo piano 2.1
Since the last update (maybe even the one before that, I'm not sure), on booting/rebooting the system the playlist is not present.
That makes the Feature "play on startup" useless. I like this feature very much.
thanks, regards

**Comment by cYCL157** [comment]
i use this feature myself and typically reboot after installing an update, including after having installed 202101021 and did not have a problem with this.  I do enjoy that the music just restarts, since i have it playing almost all of the time.  

To clarify, is the playlist cleared/empty and not just not playing?

**Reply by ghost** [reply]
This morning on booting, the playlist was empty. I just tried to replicate the issue. However now after boot the playlist was there but did not start playing. On rebooting the playlist was still there but again not playing.
On the Features page I did switch off and on the "play on startup" switch, now after reboot the playlist started playing.
Possibly a glitch from updating.
Note that on the System page the wifi was spontaniously switched on. Possibly in a strange way related with "play on startup"
For now the issue is solved.
Will keep you informed of other strange behaviours if noticed.
thanks
